pkg/virt-controller/services/net_binding.go

```diff
@@ -44,6 +45,8 @@ func NetBindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.Kube
 			if !exist {
 				return nil, fmt.Errorf("couldn't find configuration for network bindining: %s", iface.Binding.Name)
 			}
+		} else if iface.Slirp != nil {
+			pluginSidecars = append(pluginSidecars, SlirpNetBindingPluginSidecar(vmi, config, recorder))
 		}
 	}
 	for _, pluginInfo := range bindingByName {
```

Comment:
For the consistency, I would add `ImagePullPolicy: kvConfig.ImagePullPolicy` to this image as well.

Reply:
Done